src/traction_modem/Defs.hxx

```diff
@@ -56,6 +56,16 @@ struct Defs
     /// message.
     static constexpr uint16_t RESPONSE = 0x8000;
 
+    /// Normal (default) response timeout.
+    static constexpr long long RESP_TIMEOUT = SEC_TO_NSEC(3);
+
+    /// Shortened response timeout, typically used in the link establishment
+    /// phase.
+    static constexpr long long RESP_TIMEOUT_SHORT = SEC_TO_NSEC(1);
+
+    /// Time to wait for inactivity before proactively sending a ping.
```

Comment:
// This must be comfortably below RESP_TIMEOUT otherwise we will send the ping too late and consider the link to be down.